### ISSUE TITLE
Where is the callback for done of all tasks? 

### ISSUE THREAD

**Issue by trietbui85** [bitsydarel/executorservices#1]
Assuming I submit 2 tasks. Is there any callback available to catch when all tasks are finished?

**Comment by bitsydarel** [comment]
You could use Future.wait method and pass the two futures

**Comment by bitsydarel** [comment]
For your above question, there’s not such callback beside shutdown method.

**Comment by bitsydarel** [comment]
Let me rephrase it.

The executorservice's submit method return a Future so that you can track the progress of your task and get the result or  the error produced by the task.
